libs/@blockprotocol/type-system/crate/src/ontology/url/mod.rs

```diff
@@ -115,38 +114,18 @@ impl FromStr for VersionedUrl {
     type Err = ParseVersionedUrlError;
 
     fn from_str(url: &str) -> Result<Self, ParseVersionedUrlError> {
-        static RE: LazyLock<Regex> =
-            LazyLock::new(|| Regex::new(r#"(.+/)v/(\d+)(.*)"#).expect("regex failed to compile"));
-
-        if url.len() > 2048 {
```

Comment:
This should not have been removed, perhaps should happen before the split

Reply:
Oh we do this in `BaseUrl` as well.. perhaps this is fine

Reply:
I don't have a strong opinion on this, we can still have it in place as the length is different from the one in `BaseUrl`

Reply:
Let's add it because it's technically what the spec suggests

Reply:
Can we re-add it?

Reply:
Done